src/test/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/autoimport/ImporteringRepositoryTest.java

```diff
@@ -92,6 +92,7 @@ public void setUp() {
                 årstall,
                 kvartal,
                 "987654321",
+                "A",
```

Comment:
Kunne være flott å ha konstanter som forklarer disse verdier for varighet et sted.